lib/logger/utils.ts

```diff
@@ -206,3 +206,46 @@ export function withSanitizer(streamConfig: bunyan.Stream): bunyan.Stream {
 
   throw new Error("Missing 'stream' or 'path' for bunyan stream");
 }
+
+/**
+ * A function that terminates exeution if the log level that was entered is
+ *  not a valid value for the bunyan logger.
+ * @param logLevelToCheck
+ * @returns returns undefined when the logLevelToCheck is valid. Else it stops execution.
+ */
+export function validateLogLevel(
+  logLevelToCheck: string | undefined | bunyan.LogLevel
+): void {
+  const allowedValues: bunyan.LogLevel[] = [
+    'trace',
+    'debug',
+    'info',
+    'warn',
+    'error',
+    'fatal',
+  ];
+  if (!logLevelToCheck) {
+    return;
+  }
+  if (typeof logLevelToCheck === 'string') {
+    logLevelToCheck = logLevelToCheck.toLocaleLowerCase();
+    if (
+      allowedValues.includes(logLevelToCheck as bunyan.LogLevel) ||
+      logLevelToCheck.trim().length === 0
```

Comment:
```suggestion
      allowedValues.includes(logLevelToCheck as bunyan.LogLevel)
```
This would fail later, as i think bunyan don't accept ` ` or ` debug`. 

So we should always trim or never accept those values. I prefer to not accept.

Reply:
Yes but after the function call I see we are taking care of the case when `process.env=''`. Should we ignore the case when that field only has white spaces ?